engine/src/main/java/org/terasology/engine/logic/players/PlayerSystem.java

```diff
@@ -155,7 +155,7 @@ public void onConnect(ConnectedEvent connected, EntityRef entity) {
     private void restoreCharacter(EntityRef entity, EntityRef character) {
 
         Client clientListener = networkSystem.getOwner(entity);
-        LOGGER.info(clientListener.toString());
+        LOGGER.atInfo().addArgument(() -> clientListener.toString()).log("{}");
```

Comment:
I don't think we need fluent API here. The following should suffice:
```suggestion
        LOGGER.info("{}", clientListener);
```